Monika-After-Story/MonikaModDev
# Soft eyes

#3705 

wait for #4582 before testing

# Key Changes
* soft eyes from @Professor-JMO

# Testing
* verify soft eyes look okay in expreviewer
* verify soft eyes can be selected in sprite creation tool
